src/interface/src/app/backend-constants.ts

```diff
@@ -0,0 +1,3 @@
+export class BackendConstants {
+  public static Host: string = "http://localhost:8000"
```

Comment:
Can add readonly

---
if you use const assertion as suggested above, don't need to add readonly

Reply:
cool : )
I've used the suggested const assertion.